inlong-manager/manager-dao/src/main/resources/mappers/StreamSourceEntityMapper.xml

```diff
@@ -498,6 +523,38 @@
             </if>
         </where>
     </update>
+    <update id="restoreStatusFromHeartbeatByIds">
```

Comment:
Maybe call `rollbackTimeoutStatusByIds`, and no need transfer the `status` param is better.

Reply:
done